File: source_code/datn_be/src/main/java/com/findJob/datn_be/dto/response/ApplicantDetailResponse.java
package com.findJob.datn_be.dto.response;

import com.findJob.datn_be.model.student.StudentCV;
import lombok.Data;

@Data
public class ApplicantDetailResponse {
    private Long applyId;
    private Long businessId;
    private String studentEmail;
    private Integer applyStatus;

    // job
    private JobResponse jobResponse;

    // student
    private StudentCV studentCV;

    // kinh nghiem lam viec
}

File: source_code/datn_fe2/src/types/response/ApplicantDetailResponse.ts
import { JobResponse } from "./JobResponse"
import { StudentCV } from "./StudentCV"

export type ApplicantDetailResponse = {
    applyId: number,
    businessId: number,
    studentEmail: string,
    applyStatus: number,

    jobResponse: JobResponse,
    studentCV: StudentCV,
}